Gemfile

```diff
@@ -1,5 +1,8 @@
 source 'https://rubygems.org'
 
+# Use kaminari for pagination
+gem 'kaminari'
```

Comment:
Can we remove this from here and add it as a dependency in the `.gemspec`, please? I think you'll need to add a `require 'kaminari'` in the `engine.rb` file too. 

This seems to work in the dummy app, but it's not working for me in a separate, external rails app that I'm testing in. That doesn't get kaminari on bundle.